app/src/pages/inside/settingsPage/settingsTabs/autoAnalysisTab/autoAnalysisTab.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import { connect } from 'react-redux';
+import { injectIntl } from 'react-intl';
+import { fetchProjectAction, fetchAutoAnalysisConfigurationAction } from 'controllers/project';
+import { activeProjectSelector } from 'controllers/user';
+import { AnalysisForm } from './analysisForm/analysisForm';
+import { IndexActionsBlock } from './indexActionsBlock';
+import styles from './autoAnalysisTab.scss';
+
+const cx = classNames.bind(styles);
+
+@connect(
+  (state) => ({
+    projectId: activeProjectSelector(state),
+  }),
+  {
+    fetchProjectAction,
+    fetchAutoAnalysisConfigurationAction,
+  },
+)
+@injectIntl
+export class AutoAnalysisTab extends PureComponent {
```

Comment:
Connected component already pure. Do not use PureComponents without justification. Check all cases.

Reply:
Fixed